tests/lib/snaps/test-snapd-uhid/bin/create-device

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import os
+import struct
+
```

Comment:
We can ctypes and ctypes.Structure to make this code easier to read:

A quick random example from one of my throwaway projects: https://github.com/zyga/libpci/blob/master/libpci/_types.py#L83

You can couple that with attribute `_pack_` to make sure it's exactly as you specified: https://docs.python.org/3/library/ctypes.html#ctypes.Structure._pack_

Then you can write such things directly and they will be identical to that big inline pack below.